src/EFTools/EntityDesignPackage/EntityDesignPackage.csproj

```diff
@@ -66,11 +69,15 @@
     <Reference Include="Microsoft.VisualStudio.Shell.Interop.11.0" />
     <Reference Condition="'$(VisualStudioVersion)' != '11.0' AND '$(VisualStudioVersion)' != '12.0' AND '$(VisualStudioVersion)' != '14.0'"
                Include="Microsoft.VisualStudio.Shell.Interop.15.3.DesignTime" />
+    <Reference Condition="'$(VisualStudioVersion)' == '16.0'" Include="Microsoft.VisualStudio.Shell.Interop.14.0.DesignTime, Version=14.0.0.0">
```

Comment:
I am a bit surprised that we pull the 14.0 version of the assemblies for the 16.0 version of VS? Shouldn't the major version numbers match?  

Also, is it ok to have Microsoft.VisualStudio.Shell.Interop.15.3.DesignTime (per the previous line) and Microsoft.VisualStudio.Shell.Interop.14.0.DesignTime added as references at the same time? It seems both conditions apply if the version of VS is 16.0.

Reply:
The 14.0 version is required in the fix published by the PMA team for this (which is specifically for 16.0 VS onwards). I agree that 15.3 and 14.0 is weird but does not appear to cause any problems and taking out 15.3 would require extensive re-testing.

Reply:
And note, 15.3 by itself did *not* work.